Week3/week3HomeWork/Exercise1/amazingNaji.js

```diff
@@ -0,0 +1,22 @@
+function giveCompliment(myName) {
+    const compliment = [
+        "GREAT",
+        "Awesome",
+        "Fantastic",
+        "So COOL",
+        "Excellent",
+        "the best",
+        "AMAZING",
+        "Nice",
+        "Wonderful",
+        "Superhero"
+    ];
+    let randomCopliment = Math.floor(Math.random() * 10);
+    let message = "";
```

Comment:
``
let message = "";
message = "You are " + compliment[randomCopliment] + " Today " + myName + "!";
```
Here the `let message` is a bit useless, because you might as well assign it directly, then you can do these two lines on one line:)